recipe/cmake_test/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(cf_dummy LANGUAGES C CXX)
+cmake_minimum_required(VERSION 3.12)
+# We need to specify CONFIG to make sure that CMake
+# uses protobuf-config.cmake instead of relying on its
+# own FindProtobuf.cmake that does not know about
+# abseil transitive depedendency and about C++ version
+# required, see https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/4075#issuecomment-1569242816
+find_package(Protobuf CONFIG REQUIRED)
```

Comment:
@boldorider4 here we testing validity if the find_package works and it builds correctly in tests then it means is valid.